doc/ref_arch/openstack/chapters/chapter04.md

```diff
@@ -1,82 +1,209 @@
+Chapter 3 presented the core OpenStack services for creating an IaaS cloud.  The chapter discussed the deployment topology including distribution of the core OpenStack services among Controller and Compute nodes. Chapter 3 also presented security and LCM considerations that play a role in the deployment and management of the services. This Chapter will delve deeper into certain topics that need to be considered in creating and operating an OpenStack based IaaS cloud. The OpenStack IaaS cloud needs the physical (underlay) and the overlay networks needed for intra tenant and external (to the tenant) communications. Cloud topology related to host aggregates and availability zones, and minimal software versions for shared services (kernel, host operating system, common drivers, etc.). To round up the operationalization considerations, the chapter includes a listing of some of the requirements for Security and Life Cycle Management.
+
 
 <a name="4.2"></a>
-## 4.2 Network Topology (control plane, storage, tenant, external)
-- Physical Network = Spine / Leaf with dual homing to ToR
-- Proposed logical network layout – VLANs, overlay, FIPs, IPv6   – we will follow common vendor recommendations however we can agree on our minimum expectations
-- LBaaS v2 compliant Load Balancing
-- Neutron ML2 integration to any external SDN controller
+## 4.2 Network Topology 
+
+<a name="4.2.1"></a>
+### 4.2.1. Architectural Drivers – Requirements Traceability
+
+| Ref # | sub-category | Description |
+|--------|---------------|--------------------------------|
+| req.inf.ntw.01 | Network | The Architecture must provide virtual network interfaces to VM instances. |
+| req.inf.ntw.02 | Network | The Architecture must include capabilities for integrating SDN controllers to support provisioning of network services, from the OpenStack Neutron service, such as networking of VTEPs to the Border Edge based VRFs. |
+| req.inf.ntw.03 | Network | The Architecture must support low latency and high throughput traffic needs. |
+| req.inf.ntw.04 | Network | The Architecture should support service function chaining. |
+| req.inf.ntw.05 | Network | The Architecture must allow for East/West tenant traffic within the cloud (via tunnelled encapsulation overlay such as VXLAN or Geneve). |
+| req.inf.ntw.06 | Network | The Architecture should support Distributed Virtual Routing (DVR) to allow compute nodes to route traffic efficiently. |
+| req.inf.ntw.07 | Network | The Architecture must support network resiliency. |
+| req.inf.ntw.08 | Network | The NFVI Network Fabric should embrace the concepts of open networking and disaggregation using commodity networking hardware and disaggregated Network Operating Systems. |
+| req.inf.ntw.09 | Network | The NFVI Network Fabric should embrace open-based standards and technologies. |
+| req.inf.ntw.10 | Network | The NFVI Network Fabric must be capable of supporting highly available (Five 9’s or better) VNF workloads. |
```

Comment:
I think this should be reformulated to contain infrastructure requirements and not workload requirements.

Reply:
Added to #339

Reply:
Is this issue for RA Chapter02?

Reply:
@pgoyal01 nope, it is for RA Chapter 4.